src/flow.ts

```diff
@@ -186,7 +180,6 @@ export class Flow {
   static AnnotationVerticalJustify = AnnotationVerticalJustify;
   static ChordSymbolHorizontalJustify = ChordSymbolHorizontalJustify;
   static ChordSymbolVerticalJustify = ChordSymbolVerticalJustify;
-  static SymbolTypes = SymbolTypes;
```

Comment:
I assume this deletion is related to the next PR about ChordSymbol?

Reply:
This is right. Restored.